api/src/lib/interface.ts

```diff
@@ -265,7 +265,7 @@ export interface DeleteLinkPathParams {
     id: string;
 }
 
-export type ValidStatuses = 'DRAFT' | 'LIVE';
+export type ValidStatuses = 'DRAFT' | 'LIVE' | 'LOCKED';
```

Comment:
These statuses should match the ones in the DB. If we don't use HIDDEN, then we should remove it from schema.prisma also. We have more mismatches like this that we will need to address